platform-common/src/test/java/com/softicar/platform/common/ui/image/ImagesTest.java

```diff
@@ -62,6 +63,35 @@ public void testCountPixelsWithColor() {
 		assertEquals(3067, Images.countPixelsWithColor(image, textColor));
 	}
 
+	@Test
+	public void testIsSupportedWithMimeTypeJpg() {
+
+		assertTrue(Images.isSupported(MimeType.IMAGE_JPEG));
+		assertTrue(Images.isSupported(MimeType.IMAGE_PNG));
+		assertTrue(Images.isSupported(MimeType.IMAGE_TIFF));
+		assertFalse(Images.isSupported(MimeType.APPLICATION_PDF));
+	}
+
+	@Test
+	public void testIsSupportedWithSuffixJpg() {
```

Comment:
why is only `Jpg` in the test method name?

Reply:
Oops. Artifact.